lib/friendly_id/history.rb

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Typo?
```suggestion
```

Reply:
huh. yeah, i agree that's a fat-finger error. will fix.